xapian-core/backends/databaseinternal.cc

```diff
@@ -104,6 +104,20 @@ Database::Internal::get_wdf_upper_bound(const string & term) const
     return cf;
 }
 
+Xapian::termcount
+Database::Internal::get_uniq_terms_lower_bound() const
+{
+    throw Xapian::UnimplementedError("This backend doesn't support"
+			       "get_uniq_terms_lower_bound");
+}
+
+Xapian::termcount
+Database::Internal::get_uniq_terms_upper_bound() const
+{
+    throw Xapian::UnimplementedError("This backend doesn't support"
+			       "get_uniq_terms_upper_bound");
+}
```

Comment:
And the document length upper bound is a valid upper bound for the number of unique terms.

Reply:
Done